src/Item/PluginItem/Item.js

```diff
@@ -36,22 +43,30 @@ class Item extends Component {
     state = {
         showFooter: false,
         activeVisualization: this.props.item.type,
+        pluginIsAvailable: itemTypeMap[this.props.item.type].plugin
```

Comment:
You can use the shorthand notation like this:

pluginIsAvailable: !!itemTypeMap[this.props.item.type].plugin